code/modules/vending/cola.dm

```diff
@@ -39,7 +39,9 @@
 /obj/machinery/vending/cola/random/Initialize()
 	// No need to call parent, we're not doing anything with this machine. Just picking a new type of machine to use, spawning it and deleting ourselves.
```

Comment:
Okay, I don't know who added this, but can we undo it? It's just circumventing 1 proc call of sanity.

This and the snack machine